boltdesignsystem/bolt
# ES Modules: Build Tool / Config Updates

## Jira
N/A

## Summary
Build tool + related config updates (Browserslist + Babel) that are part of the broader [ES Module support](https://github.com/boltdesignsystem/bolt/pull/1579). work.

## Details
Because all of the updates here are conditional (based on the new `esModules` config boolean added to `.boltrc.js` — disabled by default), this particular piece of work should by relatively straightforward to roll out without introducing any issues or regressions. This new option is also disabled in the monorepo until all the other required pieces are in place (component + core updates).

## How to test
- Confirm Travis CI build passing as expected (Jest + Nightwatch tests passing)
- Review internal config / code changes
